mautic/mautic
# [UX] Improve Attachments tooltip to remove redundancy

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [x]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:

There are several tooltips that are longer than necessary and go back and forth on the same information that could be presented in a more concise way. This is one case.



#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Open Channels > Emails
3. Create new or edit
4. Go to the Advanced tab
5. Hover the mouse on the Attachment label and read the tooltip

![image](https://github.com/mautic/mautic/assets/116097999/c6e0309a-1396-423b-a0aa-f1773ed04a90)
